backend/junit.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites name="jest tests" tests="4" failures="0" errors="0" time="1.244">
+  <testsuite name="undefined" errors="0" failures="0" skipped="0" timestamp="2020-12-16T15:17:30" time="0.522" tests="3">
+    <testcase classname=" nicks with roles to be sorted are sorted successfully" name=" nicks with roles to be sorted are sorted successfully" time="0.004">
+    </testcase>
+    <testcase classname=" warning is printed if number of users differs" name=" warning is printed if number of users differs" time="0.022">
+    </testcase>
+    <testcase classname=" insert and delete nick on join/part" name=" insert and delete nick on join/part" time="0.066">
+    </testcase>
+  </testsuite>
+  <testsuite name="undefined" errors="0" failures="0" skipped="0" timestamp="2020-12-16T15:17:31" time="0.098" tests="1">
+    <testcase classname=" raw IRC message line is parsed to correct IrcMessage object" name=" raw IRC message line is parsed to correct IrcMessage object" time="0.001">
+    </testcase>
+  </testsuite>
+</testsuites>
```

Comment:
This file shouldn't be committed. Add it to the gitignore? Or shouldn't it already be in there? Maybe the path is wrong.

Reply:
removed and added into gitignore